docker-entrypoint.sh

```diff
@@ -43,6 +43,14 @@ if [[ "$1" == apache2* ]] || [ "$1" == php-fpm ]; then
 			EOF
 			chown www-data:www-data .htaccess
 		fi
+	else
+		CONTAINER_WORDPRESS_VERSION=$(php -r "include('/usr/src/wordpress/wp-includes/version.php'); echo \$wp_version;")
+		INSTALLED_WORDPRESS_VERSION=$(php -r "include('wp-includes/version.php'); echo \$wp_version;")
+		if ! [ "$CONTAINER_WORDPRESS_VERSION" == "$INSTALLED_WORDPRESS_VERSION" ]; then
```

Comment:
If we are going to add this, it really needs to be a full version comparison and then only upgrade when container version is newer.  Otherwise, if a user has WordPress set to auto-upgrade itself (which I believe is the default), then when the container restarts, they will get auto downgraded.

I wonder if this should also be behind a env flag of some sort so that users don't get unexpected upgrades when they start a new container.